### DISCUSSION TITLE
What happened to real time power monitoring

### DISCUSSION THREAD

**Discussion by glendower** [signalkraft/mypyllant-component#217] [Q&A]
I'm new to the Vaillant integration but I do remember being able to monitor the real (ish) power consumption of my heat pump when I first installed the integration.

However I noticed a few days ago that that entity (sensor.myaddress_device_0_arotherm_plus_current_power) was no longer available.

Has anybody any idea what has happened or have I done something stupid?

**Comment by dylan09** [comment]
Did you check in settings for integration? There are some options at the button of the dialog. You have to check `Fetch real time power...`

**Comment by glendower** [comment]
Yes, I saw that, I had searched for a solution for about a week and found the solution literally minutes after I posted the question lol, isn't it always the way.
Thank you anyway for taking the time to reply.

⁣Get BlueMail for Android ​
